fineract-provider/src/main/java/org/apache/fineract/infrastructure/campaigns/jobs/sendmessagetosmsgateway/SendMessageToSmsGatewayConfig.java

```diff
@@ -46,6 +48,9 @@ public class SendMessageToSmsGatewayConfig {
     private NotificationSenderService notificationSenderService;
     @Autowired
     private SmsConfigUtils smsConfigUtils;
+    @Autowired
+    @Qualifier("fineractDefaultThreadPoolTaskExecutor")
```

Comment:
I think you could create a constant on these specific bean name(s) so you don't have to put this string value everywhere.

Reply:
constants are used now